modules/drivers/bigquery-cloud-sdk/test/metabase/test/data/bigquery_cloud_sdk.clj

```diff
@@ -82,7 +82,10 @@
 
 (defmethod tx/dbdef->connection-details :bigquery-cloud-sdk
   [_driver _context {:keys [database-name]}]
-  (assoc (test-db-details) :dataset-id (test-dataset-id database-name) :include-user-id-and-hash true))
+  (assoc (test-db-details)
+         :dataset-filters-type "inclusion"
+         :dataset-filters-patterns (test-dataset-id database-name)
+         :include-user-id-and-hash true))
```

Comment:
`dataset-id` doesn't work with the `list-tables` function above, and was the old way of specifying the dataset ID in `connection-details`, now we do it with `dataset-filters-patterns` and `dataset-filters-type`.